### PR TITLE
Updates Cycle.js implementation details

### PR DESCRIPTION
**Pull request by AngeloGulina**
Cycle.js supports all kind of streams related libraries but it's now built using `xstream`(http://staltz.com/xstream/) which is not that heavy (nos as heavy as RxJS).

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -620,7 +620,7 @@ give you my opinions directly. Ready?  Here goes:
   neat.
 - __cycle:__ `cycle`'s pretty good - unlike most frameworks it lays out a clear
   architecture which helps with reasoning about it. That said, it's built on
-  `virtual-dom` and `RxJS` which are a bit heavy for my taste. `choo` works
+  `virtual-dom` and `xstream` which are a bit heavy for my taste. `choo` works
   pretty well for FRP style programming, but something like [inu][inu] might be
```
Comment (brodybits) [inline README.md]: Guys can you tell what [inu] is?

Comment (YerkoPalma) [inline README.md]: https://github.com/ahdinosaur/inu

Comment (brodybits) [inline README.md]: Thanks @YerkoPalma. Can you guys update the link when you get a chance?

Comment (YerkoPalma) [inline README.md]: The link is actually working for me, seems like github resolves the repo without need of pasting the full link

**Pull request merged**